E0252 needs to be updated to new format

From: src/test/compile-fail/double-import.rs

Error E0252 needs an updated unit test and span_labels, updating it from:

```
error[E0252]: a value named `foo` has already been imported in this module
  --> src/test/compile-fail/double-import.rs:25:5
   |
24 | use sub1::foo; //~ NOTE previous import of `foo` here
   |     --------- previous import of `foo` here
25 | use sub2::foo; //~ ERROR a value named `foo` has already been imported in this module [E0252]
   |     ^^^^^^^^^

error: requires `sized` lang_item

error: requires `sized` lang_item

error: requires `sized` lang_item
```

To:

```
error[E0252]: a value named `foo` has already been imported in this module
  --> src/test/compile-fail/double-import.rs:25:5
   |
24 | use sub1::foo; //~ NOTE previous import of `foo` here
   |     --------- previous import of `foo` here
25 | use sub2::foo; //~ ERROR a value named `foo` has already been imported in this module [E0252]
   |     ^^^^^^^^^ already imported
```

To remove the extra "error: requires `sized` lang_item", remove the:

```
#![feature(no_core)]
#![no_core]
```

Lines in the unit test.

--- a/src/librustc_resolve/lib.rs
+++ b/src/librustc_resolve/lib.rs
@@ -3367,7 +3367,11 @@ impl<'a> Resolver<'a> {
             (true, _) | (_, true) => struct_span_err!(self.session, span, E0260, "{}", msg),
             _ => match (old_binding.is_import(), binding.is_import()) {
                 (false, false) => struct_span_err!(self.session, span, E0428, "{}", msg),
-                (true, true) => struct_span_err!(self.session, span, E0252, "{}", msg),
+                (true, true) => {
+                    let mut e = struct_span_err!(self.session, span, E0252, "{}", msg);
+                    e.span_label(span, &format!("already imported"));
+                    e
+                },
                 _ => {
                     let mut e = struct_span_err!(self.session, span, E0255, "{}", msg);
                     e.span_label(span, &format!("`{}` was already imported", name));

--- a/src/test/compile-fail/double-import.rs
+++ b/src/test/compile-fail/double-import.rs
@@ -7,8 +7,6 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
-#![feature(no_core)]
-#![no_core]
 
 // This tests that conflicting imports shows both `use` lines
 // when reporting the error.
@@ -23,5 +21,6 @@ mod sub2 {
 
 use sub1::foo; //~ NOTE previous import of `foo` here
 use sub2::foo; //~ ERROR a value named `foo` has already been imported in this module [E0252]
+               //~| NOTE already imported
 
 fn main() {}

--- a/src/test/compile-fail/issue-26886.rs
+++ b/src/test/compile-fail/issue-26886.rs
@@ -11,7 +11,9 @@
 use std::sync::{self, Arc}; //~ NOTE previous import
                             //~^ NOTE previous import
 use std::sync::Arc; //~ ERROR a type named
+                    //~| NOTE already imported
 use std::sync; //~ ERROR a module named
+               //~| NOTE already imported
 
 fn main() {
 }

--- a/src/test/compile-fail/use-mod.rs
+++ b/src/test/compile-fail/use-mod.rs
@@ -15,7 +15,8 @@ use foo::bar::{
     Bar,
     self
 //~^ NOTE another `self` import appears here
-//~^^ ERROR a module named `bar` has already been imported in this module
+//~| ERROR a module named `bar` has already been imported in this module
+//~| NOTE already imported
 };
 
 use {self};

